mne/beamformer/_rap_music.py

```diff
@@ -212,7 +183,14 @@ def _compute_proj(A):
 
 @verbose
 def rap_music(
-    evoked, forward, noise_cov, n_dipoles=5, return_residual=False, verbose=None
+    evoked,
+    forward,
+    noise_cov,
+    n_dipoles=5,
+    return_residual=False,
+    *,
+    use_trap=False,
```

Comment:
I would rather have a public function called trap_music next to rap_music to make it easier to discover. It means rap_music with use_trap param should be made a private function and then trap_music public function would call _rap_music(..., use_trap=True)

Reply:
Hopefully done. I hope I made the right decision w.r.t the decorators and documentation. Also corrected a minor doc error.